pkg/features/kube_features.go

```diff
@@ -132,14 +116,6 @@ const (
 	// Enables the Azure File in-tree driver to Azure File Driver migration feature.
 	CSIMigrationAzureFile featuregate.Feature = "CSIMigrationAzureFile"
```

Comment:
Why is this not removed? Same with `CSIMigrationvSphere` below?

Reply:
Thanks for your reply!  I noticed it should be removed in 1.28. https://github.com/kubernetes/kubernetes/blob/e18fa0075754aa49d085826a2e0de96cf5fb5144/pkg/features/kube_features.go#L796